data/data-pipeline/data_pipeline/etl/score/schemas/datasets.py

```diff
@@ -16,24 +16,50 @@ class FieldType(Enum):
 class DatasetsConfig:
     @dataclass
     class Dataset:
+        """A class that defines a dataset and its load variables.
+
+        Attributes:
+            long_name (str): A human readable title for the dataset.
+            short_name (str): used to compose the short variable names for tiles/arcgis. All short variable names will be prepended
+            with the short name of the data set it comes from, i.e. `nri__ex_loss`.
+            module_name (str): A string that matches both the Python module name for the dataset and the `NAME` property on the ETL class.
+            description (str): A human readable description of the dataset.
+            load_fields (LoadField): A list of type LoadField that will drive the score ETL and side effects (tiles, downloadables).
+        """
+
         @dataclass
         class LoadField:
+            """A class to define the fields to be saved on the dataset's output.
+
+            These fields will be then imported by the score generation ETL.
+
+            Attributes:
+                short_name (str): Used in conjunction with the dataset's `short_name` for files where short names are needed.
+                df_field_name (str): Name for the field in the etl class.
+                long_name (str): Column name for the dataset's output csv.
+                field_type (FieldType): An enum that dictates what type of field this is. This will be used on the `etl_score_post`
+                for the data manipulation. The `by_value` metadata prop will load the field type's Enum value instead of the index, i.e. "string"
+                and not STRING reverse_percentile (Optional bool): An optional bool to denote this field to be a reverse_percentile.
+                TODO: data/data-pipeline/data_pipeline/etl/score/etl_score.py:477
+                include_in_tiles (bool): Include this field on the tile export.
+                include_in_csv (bool): Include this field on the CSV export.
+                include_in_excel (bool): Include this field on the Excel export.
+                column_in_position (int): column position in CSV and Excel files
```

Comment:
We might actually want to make this a float? 

That way you don't have to update every single field's order precisely as integers in order. 

E.g., it'd be easy to update: 

```
field1 column_order: 1
field2 column_order: 2
field3 column_order: 3
...
field37 column_order: 37
```

To 

```
field1 column_order: 1
field2 column_order: 2
field38 column_order: 2.5 [only value that has changed]
field3 column_order: 3
...
field37 column_order: 37
```

Which is less complicated of a change than: 

```
field1 column_order: 1
field2 column_order: 2
field38 column_order: 3 [updated]
field3 column_order: 4 [updated]
...
field37 column_order: 38 [updated]
```


Without having to re-specify the order of the other 36 fields. Does that make sense?

Reply:
Totally! Updated the code.